price-estimator/src/models.rs

```diff
@@ -102,6 +102,11 @@ impl From<&pricegraph::TransitiveOrderbook> for MarketsResult {
     }
 }
 
+/// A type representing a price estimate result.
+#[derive(Debug, Serialize)]
+#[serde(transparent)]
+pub struct PriceEstimateResult(pub Option<f64>);
```

Comment:
I assume the fact that the price is in quote is evident?

Reply:
I'll add a note in the comments somewhere.